tests/integration/hive_metastore/test_migrate.py

```diff
@@ -370,30 +370,61 @@ def test_revert_migrated_table(sql_backend, runtime_ctx, make_catalog):
 
 @retried(on=[NotFound], timeout=timedelta(minutes=5))
 def test_mapping_skips_tables_databases(ws, sql_backend, runtime_ctx, make_catalog):
```

Comment:
what's the added runtime  for this test? the more test does, the more flaky it may become, so i'm wondering if it makes sense to grow this test case or add another one. it's always a tradeoff in the integration tests - it's fine to add more "faster ones", but better to extend "slower ones".